ChangeLog.md

```diff
@@ -1,4 +1,17 @@
 ## Changelog
+
+3.3.0
+* BREAKING: shutdown() function on client now destroy the client and future requests throw error
```

Comment:
If it's breaking, why not go to 4.0.0?

Reply:
it is minor, but I believe you're right.